trueline.sh

```diff
@@ -445,6 +445,25 @@ _trueline_cmd_duration_segment() {
     fi
 }
 
+_trueline_distro_icon_segment() {
+    case "$(uname -a)" in
```

Comment:
I tried this out and `uname -a` returns linux in my case instead of the distro name (I use arch).. i guess it's better to either use `lsb_release -d` or `cat /etc/os-release` to actually get the distro name

Reply:
Hey yep apparently `uname` doesn't have the same output to other distributions. 
For example in my Fedora 38 worked.
But what ever you are correct.
So I changed it to `cat /etc/os-release | grep ^ID= `
So it will work everywhere hopefully. I don't chose the `lsb_release` because user needs to install `lsb`.